shell/connect.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+require_once 'abstract.php';
+ 
+class TBT_Shell_Platform_Connect extends Mage_Shell_Abstract
+{
+    protected $username;
+    protected $password;
+    protected $devMode;
+ 
+    public function __construct() 
+    {
+        parent::__construct();
+ 
+        $this->username = $this->getArg('username');
+        $this->password = $this->getArg('password');
+        $this->devMode = $this->getArg('devmode');
```

Comment:
We should add some validation here to make sure the username and password was received.